scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -261,14 +289,15 @@ if [[ -z "$projectName" ]]; then
 		done
 
 		init_env_file
-		create_compose_files
+		create_compose_files "$preferredRelease"
 		projects=$(get_existing_projects)
 		;;
 	esac
 
 	envCount=$(find . -name "*.env" -type f | wc -l)
 	if [ "$envCount" -gt 0 ]; then
 		while [[ -z "$projectName" ]]; do
+		  echo
```

Comment:
Not needed?

Reply:
... or new line?

Reply:
yeah - I'm going for a new line here.  each of the questions felt really cramped, especially when we're going to show a multi-column output with all the branches/versions.  Open to another way of doing this but would prefer to keep the extra spaces!

Reply:
Thanks... FYI I've just tried this script out to test a new SW library and it worked well!